<?xml version="1.0" encoding="utf-8"?>
<!-- This URDF was automatically created by SolidWorks to URDF Exporter! Originally created by Stephen Brawner (brawner@gmail.com) 
     Commit Version: 1.6.0-4-g7f85cfe  Build Version: 1.6.7995.38578
     For more information, please see http://wiki.ros.org/sw_urdf_exporter -->
<robot
  name="arm2.0">
  <link
    name="base_link">
    <inertial>
      <origin
        xyz="-0.0052782 0.42344 -0.040273"
        rpy="0 0 0" />
      <mass
        value="13.134" />
      <inertia
        ixx="0.12169"
        ixy="0.0033159"
        ixz="0.0033397"
        iyy="0.032208"
        iyz="-0.003345"
        izz="0.1213" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://arm2.0/meshes/base_link.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.79216 0.81961 0.93333 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://arm2.0/meshes/base_link.STL" />
      </geometry>
    </collision>
  </link>
  <link
    name="Link1">
    <inertial>
      <origin
        xyz="-0.0341557930468127 0.122616971089813 -1.27924045786876E-07"
        rpy="0 0 0" />
      <mass
        value="2.58255268807088" />
      <inertia
        ixx="0.0130139121861334"
        ixy="0.00405232491310445"
        ixz="-4.28450891574271E-08"
        iyy="0.00580947647637596"
        iyz="-7.10598430705665E-07"
        izz="0.0142006453008754" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://arm2.0/meshes/Link1.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.792156862745098 0.819607843137255 0.933333333333333 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://arm2.0/meshes/Link1.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="j1"
    type="revolute">
    <origin
      xyz="0 0.1845 0"
      rpy="0 0 0" />
    <parent
      link="base_link" />
    <child
      link="Link1" />
    <axis
      xyz="0 -1 0" />
    <limit
      lower="0"
      upper="0"
      effort="0"
      velocity="0" />
  </joint>
  <link
    name="Link2">
    <inertial>
      <origin
        xyz="0.032479 0.20308 -6.2014E-05"
        rpy="0 0 0" />
      <mass
        value="3.926" />
      <inertia
        ixx="0.079554"
        ixy="0.0017335"
        ixz="3.1679E-06"
        iyy="0.0069392"
        iyz="-2.3436E-05"
        izz="0.081061" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://arm2.0/meshes/Link2.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.79216 0.81961 0.93333 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://arm2.0/meshes/Link2.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="j2"
    type="revolute">
    <origin
      xyz="-0.000250000000000007 0.1785 0"
      rpy="0 0 0" />
    <parent
      link="Link1" />
    <child
      link="Link2" />
    <axis
      xyz="1 0 0" />
    <limit
      lower="0"
      upper="0"
      effort="0"
      velocity="0" />
  </joint>
  <link
    name="Link3">
    <inertial>
      <origin
        xyz="-0.0291986987675039 -0.0483462747273982 -0.0863422706938064"
        rpy="0 0 0" />
      <mass
        value="1.93910828514451" />
      <inertia
        ixx="0.0107111547199636"
        ixy="0.0016483172358137"
        ixz="0.00333306856931167"
        iyy="0.0103103781174519"
        iyz="-0.00336693383921128"
        izz="0.00560839682066395" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://arm2.0/meshes/Link3.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.792156862745098 0.819607843137255 0.933333333333333 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://arm2.0/meshes/Link3.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="j3"
    type="revolute">
    <origin
      xyz="0 0.389999999999999 0"
      rpy="0 0 0" />
    <parent
      link="Link2" />
    <child
      link="Link3" />
    <axis
      xyz="-1 0 0" />
    <limit
      lower="0"
      upper="0"
      effort="0"
      velocity="0" />
  </joint>
  <link
    name="Link4">
    <inertial>
      <origin
        xyz="-0.0319603196677878 -0.130326564223853 -3.1956051910198E-07"
        rpy="0 0 0" />
      <mass
        value="1.48564830223244" />
      <inertia
        ixx="0.0105102634532792"
        ixy="-0.00410287807559147"
        ixz="1.92797036805706E-08"
        iyy="0.00335669837032608"
        iyz="5.33092635492305E-08"
        izz="0.0125216631987523" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://arm2.0/meshes/Link4.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.792156862745098 0.819607843137255 0.933333333333333 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://arm2.0/meshes/Link4.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="j4"
    type="revolute">
    <origin
      xyz="0.000250000000000423 -0.159242809429263 -0.149976330526953"
      rpy="0 0 0" />
    <parent
      link="Link3" />
    <child
      link="Link4" />
    <axis
      xyz="0 -1 0" />
    <limit
      lower="0"
      upper="0"
      effort="0"
      velocity="0" />
  </joint>
  <link
    name="Link5">
    <inertial>
      <origin
        xyz="-0.00390855102788603 0.0050554928508631 4.67688145711076E-08"
        rpy="0 0 0" />
      <mass
        value="0.789456345932932" />
      <inertia
        ixx="0.00139383729966315"
        ixy="-1.46027496755452E-05"
        ixz="2.62607712978393E-09"
        iyy="0.00078077007989036"
        iyz="-7.32441707362034E-10"
        izz="0.00146715996873719" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://arm2.0/meshes/Link5.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.792156862745098 0.819607843137255 0.933333333333333 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://arm2.0/meshes/Link5.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="j5"
    type="revolute">
    <origin
      xyz="0 -0.2665 0"
      rpy="0 0 0" />
    <parent
      link="Link4" />
    <child
      link="Link5" />
    <axis
      xyz="-1 0 0" />
    <limit
      lower="0"
      upper="0"
      effort="0"
      velocity="0" />
  </joint>
  <link
    name="Link6">
    <inertial>
      <origin
        xyz="0.000639113652415819 0.0107736134383868 4.70014921391537E-05"
        rpy="0 0 0" />
      <mass
        value="0.141894854238648" />
      <inertia
        ixx="7.00623194753971E-05"
        ixy="2.98477825869023E-08"
        ixz="2.7709265395048E-07"
        iyy="0.000120209877277034"
        iyz="-8.24265766433737E-09"
        izz="6.76625055908213E-05" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://arm2.0/meshes/Link6.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.792156862745098 0.819607843137255 0.933333333333333 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://arm2.0/meshes/Link6.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="j6"
    type="revolute">
    <origin
      xyz="-0.000250000000446747 -0.0839999999999992 0"
      rpy="0 0 0" />
    <parent
      link="Link5" />
    <child
      link="Link6" />
    <axis
      xyz="0 -1 0" />
    <limit
      lower="0"
      upper="0"
      effort="0"
      velocity="0" />
  </joint>
</robot>